backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -361,6 +367,18 @@ public Person addPatient(
     return savedPerson;
   }
 
+  /** This method associates PhoneNumbers with a new patient and ensures there are no duplicates */
+  public List<PhoneNumber> assignPhoneNumbersToPatient(Person person, List<PhoneNumber> incoming) {
+    if (incoming == null) {
+      return null;
```

Comment:
Sonar would prefer that this return an empty List instead. I think I agree - the way you have it written right now there likely won't be any NPEs, but it's still safer to avoid returning `null` in the long run.